src/reducers/starWarsReducer.js

```diff
@@ -1,10 +1,38 @@
-import /* we need our action types here*/ "../actions";
+import {
+  FETCH_API_START,
+  FETCH_API_SUCCESS,
+  FETCH_API_ERROR
+} from "../actions";
+
 const initialState = {
-  characters: []
+  characters: [],
+  error: " ",
```

Comment:
```suggestion
  error: "",
```
Ok yesterday I got mixed up, empty strings are falsy, but with that space in there it'll evaluate to truthy, so make sure those quotes are empty